mmv1/products/datastream/Stream.yaml

```diff
@@ -764,6 +784,211 @@ properties:
             default_from_api: true
             validation: !ruby/object:Provider::Terraform::Validation
               function: 'validation.IntAtLeast(0)'
+      - !ruby/object:Api::Type::NestedObject
+        name: 'sqlServerSourceConfig'
+        min_version: beta
```

Comment:
```suggestion
```

Reply:
Removed in https://github.com/GoogleCloudPlatform/magic-modules/commit/939b4f1e8962c3ac939f24d2fc98170f48bdc4d7